src/style.css

```diff
@@ -1 +1,77 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body,
+ul,
+li,
+nav {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.header {
+  min-width: 1024px;
+  height: 60px;
```

Comment:
don't set the height to the header. set it to link, then you won't need to set 100% to all descendants